opentelemetry-gigwarm-exporter/gigwarm-uploader-ffi/Cargo.toml

```diff
@@ -0,0 +1,7 @@
+[package]
+name = "gigwarm-uploader-ffi"
+version = "0.1.0"
+edition = "2021"
+
+[dependencies]
+
```

Comment:
can you add to all of the new `Cargo.toml` at the end of the file

```
[lints]
workspace = true
```

I'll see if I can add a CI check, which makes sure it is not forgotten 🧐